bb/bb.go

```diff
@@ -336,13 +327,20 @@ func oneCmd(c Command) {
 		}
 	}
 	initMap += "\n\t\"" + c.CmdName + "\":" + c.CmdName + "Init,"
+	// In the bb case, the commands are built. In some cases, we want to
+	// specify init= for a u-root command on boot. Hence, it now makes sense
```

Comment:
How are you specifying init=/ubin/date for example? It can't be the kernel cmdline right? doesn't the kernel cmdline get ignored if it finds a /init in the VFS?